rllib/algorithms/algorithm_config.py

```diff
@@ -427,6 +405,14 @@ def __init__(self, algo_class: Optional[type] = None):
         # `forward_exploration` to specify custom exploration behavior.
         self.exploration_config = {}
 
+        # `self.api_stack()`
+        self.enable_rl_module_and_learner = True
+        self.enable_env_runner_and_connector_v2 = True
+        self.api_stack(
```

Comment:
We have to call this method here (unfortunately) to make sure a possibly old API stack `exploration_config` gets properly saved (and then restored if necessary).